tekton/src/tasks/fvt/fvt-manage-cypress.yml.j2

```diff
@@ -0,0 +1,204 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mas-fvt-manage-cypress
```

Comment:
I can't find where this task is referenced anywhere Maeve?
(Just trying to understand the flow for my sins)